lib/srv/discovery/status.go

```diff
@@ -45,6 +45,12 @@ import (
 // - AWS Sync (TAG) status
 // - AWS EC2 Auto Discover status
 func (s *Server) updateDiscoveryConfigStatus(discoveryConfigName string) {
+	// Static configurations (ie those in `teleport.yaml/discovery_config.<cloud>.matchers`) do not have a DiscoveryConfig resource.
```

Comment:
I forgot to ask in the original PR: isn't that `teleport.yaml/discovery_service`?

Reply:
Nice catch 🙏 
Yes, it should be.
I'll create another PR for the fix because we already merged it in `master`